hysterical-horses/code_jam/dashboard/templates/dashboard/base.html

```diff
@@ -3,6 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <title>Hysterical Horses</title>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```

Comment:
Needed to shift to the top for the jQuery packages to load before the jQuery scripts were ran. Didn't work otherwise.

Reply:
Ty for letting me know, I was curious about this.